hummingbot/client/hummingbot_application.py

```diff
@@ -59,6 +62,7 @@ def logger(cls) -> HummingbotLogger:
         global s_logger
         if s_logger is None:
             s_logger = logging.getLogger(__name__)
+            s_logger = logging.getLogger(str(cls.uid))
```

Comment:
Don't think we want to do this here unless mqtt_bridge? is enabled

Reply:
uid exists for bots with or without mqtt enabled. Though L64 should definitely be removed :)
I will prepare a fix.

Cheers @TheHolyRoger !